lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -302,8 +371,10 @@ function clear(callback) {
 
 exports.create = alarmsInt(constants.MONGO_ALARM + '_01', intoTrans(context, createGroup));
 exports.list = alarmsInt(constants.MONGO_ALARM + '_02', intoTrans(context, listGroups));
+exports.listConfigGroups = alarmsInt(constants.MONGO_ALARM + '_12', intoTrans(context, listConfigGroups));
 exports.init = alarmsInt(constants.MONGO_ALARM + '_03', intoTrans(context, init));
 exports.find = alarmsInt(constants.MONGO_ALARM + '_04', intoTrans(context, find));
+exports.findConfigGroups = alarmsInt(constants.MONGO_ALARM + '_13', intoTrans(context, findConfigGroups));
 exports.findType = alarmsInt(
```

Comment:
Exporting duplicated functions, not required

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11